01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,23 @@
+const { writeFile, readFile } = require("fs").promises;
+
+async function writer() {
+    try {
+    await writeFile("temp.txt","Line 1\nLine 2\nLine 3");
```

Comment:
Ideally you would return the await writeFile and have two separate writeFile statements. You don't have to correct but good thing to note.

 await writeFile("temp.txt","Line 1");
  await writeFile("temp.txt","Line 2\n");
   var test = await writeFile("temp.txt","Line 3");
   return test // where test would be a promise